11ty/definitions/idiot.md

```diff
@@ -1,8 +1,24 @@
 ---
 title: Idiot
```

Comment:
lowercase i*

(Admittedly I've been inconsistent about managing this, so we have a lot of capitalised words that shouldn't be!)